tmt/steps/provision/__init__.py

```diff
@@ -524,14 +526,18 @@ def reboot(self, hard=False):
 
     def reconnect(self):
         """ Ensure the connection to the guest is working after reboot """
+        # Try to wait for machine to really shutdown sshd
+        time.sleep(SSH_INITIAL_WAIT_TIME)
         self.debug("Wait for a connection to the guest.")
         for attempt in range(1, CONNECTION_TIMEOUT):
+            # It's here because we allow machine to kill sshd in 2 seconds
+            # window
+            time.sleep(1)
```

Comment:
I'm not sure about this: So we first sleep for 5 seconds and then again sleep for 1 second to give sshd 2 seconds? Would it be better to put the `sleep(1)` at the end or under the `except` clause?

Reply:
Old commend. Removed. Thanks.
I just give sshd one more second to die.
Now, it's wait for 6 seconds. Not sure why it's better to hide sleep inside exception path :/.

Reply:
I thought this 1 second sleep is the space between attempts, that's why it seems a bit strange at the beginning of the loop. But nothing fatal, if you think 6 seconds for sshd is better, then let's keep it.

Please, rebase on the latest master, so that tests are not affected by the packit issue. Thanks.